src/main/groovy/org/shipkit/gradle/ReleaseConfiguration.java

```diff
@@ -52,6 +52,7 @@ public ReleaseConfiguration() {
     //TODO currently it's not clear when to use class fields and when to use the 'configuration' map
     //Let's make it clear in the docs
     private boolean dryRun = true;
+    private boolean releaseAllJavaModules = true;
```

Comment:
Fine with me, although shipkit.publishAllJavaSubprojects may be a little more intuitive, since "subproject" is a gradle term and there is a well-known "maven-publish" plugin.